lib/scoped_search/validators.rb

```diff
@@ -0,0 +1,7 @@
+module ScopedSearch
```

Comment:
You are right. I keep forgetting that it's not a rails project - no autoloading.

Updated the PR accordingly.